http_check/datadog_checks/http_check/data/conf.yaml.example

```diff
@@ -358,3 +358,9 @@ instances:
     ## Whether or not to debug log the HTTP(S) requests made, including the method and URL.
     #
     # log_requests: false
+
+    ## @param min_collection_interval - integer - optional
+    ## Specifies the minimum time elapsed between http calls from this instance, in seconds
+    ## See https://docs.datadoghq.com/developers/write_agent_check/?tab=agentv6v7#collection-interval for more details
+
+    # min_collection_interval: 30
```

Comment:
Default should be 15

https://github.com/DataDog/integrations-core/blob/286a328abc4e78e6ea2e9455fc29df3c7054b4aa/consul/datadog_checks/consul/data/conf.yaml.example#L376-L380

Reply:
Thank you, @ChristineTChen! I will update the wording